### PR TITLE
Revised breaking changes for structs in .NET Core 2.1

### PR DESCRIPTION
**Pull request by pkulikov**
Also, added the link to breaking changes to the page about What's new in .NET Core 2.1.

### CODE REVIEW

File: includes/core-changes/corefx/2.1/instantiate-struct.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ### Private fields added to built-in struct types
 
-Private fields were added to built-in struct types in [reference assemblies](../../../../docs/standard/assembly/reference-assemblies.md). As a result, in C#, struct types must always be instantiated by using the [new operator](../../../../docs/csharp/language-reference/operators/new-operator.md) or [default literal](../../../../docs/csharp/language-reference/operators/default.md#default-literal), or by initializing each of the private fields.
```
Comment (pkulikov) [inline includes/core-changes/corefx/2.1/instantiate-struct.md]: As a user of a struct, I have no means to initialize its private fields, except calling a constructor.

File: includes/core-changes/corefx/2.1/instantiate-struct.md [hunk 2]
```diff
@@ -43,16 +43,6 @@ if (key.ToString() == "y")
...
 
-```csharp
-ConsoleKeyInfo[] keys = new ConsoleKeyInfo[5];    // Array of structs.
-
-for (int i = 0; i < keys.Length; i++)
-{
-    // Initialize each array element with the new operator.
-    keys[i] = new ConsoleKeyInfo();
-}
-```
```
Comment (pkulikov) [inline includes/core-changes/corefx/2.1/instantiate-struct.md]: The following code compiles in .NET Core 3.1, so no need to initialize each array element separately:
```csharp
ConsoleKeyInfo[] keys = new ConsoleKeyInfo[5];
for (int i = 0; i < keys.Length; i++)
{
    Console.WriteLine(keys[i]);
}
```

As far as I understand, the elements of a struct array are initialized into the default value of the struct (as the result of allocating zeroed memory for an array and zero-bit patterns being the default value of structs).

**Review by tdykstra — APPROVED** [review]
LGTM

**Review by Thraka — APPROVED** [review]

**Comment by pkulikov** [comment]
@tdykstra @Thraka could anyone of you merge this PR, please.

**Pull request merged**